src/Data/pisaClient.ts

```diff
@@ -1,9 +1,15 @@
-import { createRestApiClient, currentLanguage, load } from 'scrivito'
-import { ensureString } from '../utils/ensureString'
+import { Obj, createRestApiClient, currentLanguage, load } from 'scrivito'
+import { isHomepage } from '../Objs/Homepage/HomepageObjClass'
 
 export async function pisaUrl(): Promise<string> {
-  const url = ensureString(import.meta.env.PISA_URL)
-  if (!url) throw new Error('PISA_URL is not set!')
+  const defaultRoot = await load(() => Obj.onSite('default').root())
+  if (!isHomepage(defaultRoot)) return never()
+
+  const url = defaultRoot.get('pisaUrl')
+  if (!url) {
+    console.log('Please configure a pisaUrl on the default homepage.')
```

Comment:
* [ ] FYI Not sure if this is helpful. It spams the console which typically is invisible to editors while the fix needs content changes. Would a validation make more sense?

Reply:
I think this is helpful. Ideally you only have to setup the pisaUrl once - when you initialize the tenant. Validations are not ideal, because the would only be executed, if something changed on the homepage - or may even prevent the initial content script from working properly.

This console log is intended for a developer who tries to find out, why the app is not loading e.g. the current user.